lib/pact/consumer/server.rb

```diff
@@ -66,7 +66,11 @@ def responsive?
     end
 
     def run_default_server(app, port)
-      require 'rack/handler/webrick'
```

Comment:
Given that we have rackup declared as a dependency now, do we need to use this approach, or could we go straight to requiring rackup?